### PR TITLE
Added admin UI for project approval

### LINKED ISSUE #1575: Add Admin UI for project approval
# Problem

We need an Admin UI to view projects and approve them

## Subtasks
- [x] Add projects tab
- [x] Lists all projects, each linking to their show page
- [x] Unapproved projects have an "approve" button

### PR DESCRIPTION
**Pull request by begedin**
# What's in this PR?

Closes #1575 

Still in the process of wrapping up, but ought to be ok for review.

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ Router.map(function() {
     this.route('organization-invites', function() {
       this.route('new');
     });
+    this.route('projects', function() {});
```
Comment (joshsmith) [inline app/router.js]: I think the `function` is unnecessary here.

File: app/routes/admin/projects/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  model() {
+    return this.store.findAll('project');
```
Comment (joshsmith) [inline app/routes/admin/projects/index.js]: This is fine for now, though we'll need to eventually paginate it.

File: app/router.js (hunk 1 above)
Comment (begedin) [reply app/router.js]: I've been meaning to comment on that.

Without the `function`, there's no `projects.index` route being autogenerated, so we only get the `projects` route, meaning the file structure is not consistent with github events and organization invites.

Really, that would be fine, but my assumption is, we'll soon have other subroutes of projects anyway, so leaving this here might save us some time.

I could

- leave this here
- leave this here and add a comment to explain why it's here
- remove this and 
  - move `routes/admin/projects/index.js` -> `/routes/admin/project.js/`
  - move `controllers/admin/projects/index.js` -> `/controllers/admin/project.js/`
  - move `tests/pages/admin/projects/index.js` -> `tests/pages/admin/projects.js`

File: app/routes/admin/projects/index.js (hunk 2 above)
Comment (begedin) [reply app/routes/admin/projects/index.js]: Probably, same as with org invites, albeit probably sooner than that.

File: app/router.js (hunk 1 above)
Comment (joshsmith) [reply app/router.js]: I think fine leaving it for now.

File: app/templates/admin/projects/index.hbs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        {{else if project.approvalRequested}}
+          {{fa-icon "question-circle"}} Pending approval
+        {{else}}
+          {{fa-icon "asterisk"}} Newly created
+        {{/if}}
+      </span>
+      <span data-test-actions class="log-cell log-cell--shrink">
+        {{#if project.approvalRequested}}
+          <button {{action approve project}} class="default small">Approve</button>
+        {{/if}}
```
Comment (joshsmith) [inline app/templates/admin/projects/index.hbs]: This logic means that you see the approval button simply if `project.approvalRequested` is `true`, but will still show if `project.approved` is `true`.

**Pull request merged**